internal/conf/configuration.go

```diff
@@ -102,14 +102,29 @@ type JWTConfiguration struct {
 	KeyID            string   `json:"key_id" split_words:"true"`
 }
 
+type MFAFactorTypeConfiguration struct {
+	EnrollEnabled bool `split_words:"true" default:"true"`
+	VerifyEnabled bool `split_words:"true" default:"true"`
+}
+
+type SMSFactorTypeConfiguration struct {
+	MFAFactorTypeConfiguration
+	OtpLength    int                `json:"otp_length" split_words:"true"`
+	SMSTemplate  *template.Template `json:"-"`
+	MaxFrequency time.Duration      `json:"max_frequency" split_words:"true"`
+	Template     string             `json:"template"`
```

Comment:
what's the difference between SMSTemplate and Template here?

Reply:
seems like if SMSTemplate is missing, it results in an unexpected failure - maybe worth adding a default value for it? it can be the same as the default template for the phone signups

Reply:
oops looks like it's already handled here - https://github.com/supabase/auth/pull/1668/files#diff-4c28cb40881781a1067b3b3681c43f805dab629f31f3c7614b0f781ffa096505R709-R720

Reply:
Template is defined as an env variable which we use to create the `SMSTemplate`. Should mirror the Phone Config I think but lmk if there are remaining concerns